### PR TITLE
LEAN-4095:  Support the @fig-type attribute for the <fig> element

### PR DESCRIPTION
**Pull request by mbetamony**

### CODE REVIEW

File: src/schema/nodes/figure_element.ts [hunk 1]
```diff
@@ -35,6 +38,7 @@ export const figureElement: NodeSpec = {
     id: { default: '' },
     attribution: { default: undefined },
     dataTracked: { default: null },
+    position: { default: undefined },
```
Comment (mnatsheh) [inline src/schema/nodes/figure_element.ts:41]: Based on my comment in the ticket, let's change this to "type".

Comment (mbetamony) [reply src/schema/nodes/figure_element.ts:41]: We currently export the figure node as a figure element with  `fig-type = figure`, and the  listing node as a figure element with the `fig-type = listing`. We're not going to do that anymore?

Comment (mnatsheh) [reply src/schema/nodes/figure_element.ts:41]: Yeah we're likely going to stop that. We're getting rid of listings at some point, so how we export them now doesn't matter much.

Comment (mbetamony) [reply src/schema/nodes/figure_element.ts]: Got it, done.

**Review by mnatsheh — APPROVED** [review]

**Pull request merged**